src/e3/electrolyt/plan.py

```diff
@@ -64,6 +117,20 @@ def __init__(
         for name, cls in entry_point_cls.items():
             self.mod.__dict__[name] = partial(entry_point, self.entry_points, cls, name)
 
+        self.plan_date = datetime.now(timezone.utc)
+        self.mod.__dict__["cond"] = self.cond
+
+    def cond(self, name: str, date: Callable[[datetime], bool]) -> ConditionalConstant:
```

Comment:
Can you adjust a bit the code to accept for date a boolean ? We might still need that. In that case you may rename the parameter

Reply:
As discussed live we don't know whether there is a need for now. So we'll keep the code as it is.